cmplibrary/src/main/java/com/sourcepoint/cmplibrary/SpConsentLibImpl.kt

```diff
@@ -415,12 +418,26 @@ internal class SpConsentLibImpl(
     }
 
     override fun dispose() {
+        webview = null
         executor.dispose()
         viewManager.removeAllViews()
     }
 
     private fun logMess(mess: String) = pLogger.d(this::class.java.simpleName, mess)
 
+    override fun onBackPressed() {
+        webview.let {
+            (webview as ConsentWebView).evaluateJavascript(
+                """
+                window.postMessage({ name: 'sp.BACK' })
+                """.trimIndent(),
+                null
+            )
+        }
+    }
+
+    override fun isWebviewShown(): Boolean = (webview as ConsentWebView).isShown
```

Comment:
i think here we can add an extra safety check, something like `(webview as? ConsentWebView)?.isShown ?: false`, since `webview` here might be nullable, so extra safety is never an overkill